packages/graphql/tests/tck/issues/1364.test.ts

```diff
@@ -147,8 +148,9 @@ describe("https://github.com/neo4j/graphql/issues/1364", () => {
                 RETURN count(DISTINCT genre)\\", { this: this, auth: $auth }) AS this_totalGenres
                 RETURN head(collect(this_totalGenres)) AS this_totalGenres
             }
+            WITH *
+            ORDER BY this_totalGenres ASC
```

Comment:
Does it make sense to add a test when 2 cypher fields are sorted? And the combination of cypher and normal fields?

Reply:
the second case you mention is already covered In #2578 

The case of 2 cyphers is not covered and I agree it'll be a good idea